nextflow.config

```diff
@@ -352,30 +352,73 @@ set -C # No clobber - prevent output redirection from overwriting files.
 // Disable process selector warnings by default. Use debug profile to enable warnings.
 nextflow.enable.configProcessNamesValidation = false
 
-def trace_timestamp = new java.util.Date().format( 'yyyy-MM-dd_HH-mm-ss')
 timeline {
     enabled = true
-    file    = "${params.outdir}/pipeline_info/execution_timeline_${trace_timestamp}.html"
+    file    = "${params.outdir}/pipeline_info/execution_timeline_${params.trace_report_suffix}.html"
 }
 report {
     enabled = true
-    file    = "${params.outdir}/pipeline_info/execution_report_${trace_timestamp}.html"
+    file    = "${params.outdir}/pipeline_info/execution_report_${params.trace_report_suffix}.html"
 }
 trace {
     enabled = true
-    file    = "${params.outdir}/pipeline_info/execution_trace_${trace_timestamp}.txt"
+    file    = "${params.outdir}/pipeline_info/execution_trace_${params.trace_report_suffix}.txt"
 }
 dag {
     enabled = true
-    file    = "${params.outdir}/pipeline_info/pipeline_dag_${trace_timestamp}.html"
+    file    = "${params.outdir}/pipeline_info/pipeline_dag_${params.trace_report_suffix}.html"
 }
 
 manifest {
     name            = 'nf-core/taxprofiler'
-    author          = """James A. Fellows Yates, Sofia Stamouli, Moritz E. Beber, Lili Andersson-Li, and the nf-core/taxprofiler team"""
+    author          = """James A. Fellows Yates, Sofia Stamouli, Moritz E. Beber, Lili Andersson-Li, and the nf-core/taxprofiler team""" // The author field is deprecated from Nextflow version 24.10.0, use contributors instead
+    contributors    = [
+        // TODO nf-core: Update the field with the details of the contributors to your pipeline. New with Nextflow version 24.10.0
+        [
+            name: 'James A. Fellows Yates',
+            affiliation: 'Leibniz Institute for Natural Product Research and Infection Biology - Hans Knöll Institute; Max Planck Institute for Evolutionary Anthropology, Leipzig',
+            email: '',
+            github: 'https://github.com/jfy133',
+            contribution: ['author', 'maintainer'], // List of contribution types ('author', 'maintainer' or 'contributor')
+            orcid: 'https://orcid.org/0000-0001-5585-6277'
+        ],
+        [
+            name: ' Sofia Stamouli',
+            affiliation: 'Karolinska Institutet; Karolinska University Hospital; Clinical Genomics SciLifeLab, Solna',
+            email: '',
+            github: 'https://github.com/sofstam',
+            contribution: ['author', 'maintainer'], // List of contribution types ('author', 'maintainer' or 'contributor')
+            orcid: 'https://orcid.org/0009-0006-0893-3771'
+        ],
+        [
+            name: ' Moritz E. Beber',
+            affiliation: 'Unseen Bio ApS, Copenhagen​',
+            email: '',
+            github: 'https://github.com/Midnighter',
+            contribution: ['author', 'maintainer'], // List of contribution types ('author', 'maintainer' or 'contributor')
```

Comment:
I guess @Midnighter doesn't have so much time for maintaining the pipeline atm (but leave that up for him)